LICENSE

```diff
@@ -45,22 +45,22 @@ Copyright (c) 2016      ARM, Inc.  All rights reserved.
 Copyright (c) 2010-2011 Alex Brick <bricka@ccs.neu.edu>.  All rights reserved.
 Copyright (c) 2012      The University of Wisconsin-La Crosse. All rights reserved.
 Copyright (c) 2013-2020 Intel, Inc. All rights reserved.
-Copyright (c) 2011-2021 NVIDIA Corporation.  All rights reserved.
+Copyright (c) 2011-2023 NVIDIA Corporation.  All rights reserved.
 Copyright (c) 2016-2018 Broadcom Limited.  All rights reserved.
-Copyright (c) 2011-2021 Fujitsu Limited.  All rights reserved.
+Copyright (c) 2011-2023 Fujitsu Limited.  All rights reserved.
 Copyright (c) 2014-2015 Hewlett-Packard Development Company, LP.  All
                         rights reserved.
-Copyright (c) 2013-2021 Research Organization for Information Science (RIST).
+Copyright (c) 2013-2023 Research Organization for Information Science (RIST).
                         All rights reserved.
-Copyright (c) 2017-2021 Amazon.com, Inc. or its affiliates.  All Rights
+Copyright (c)           Amazon.com, Inc. or its affiliates.  All Rights
```

Comment:
Ummm...why are there no dates associated with Amazon any more? And did HPE, Fujitsu, Sandia, and Google actually submit changes this year?

Reply:
I followed #9039 by bumping all '21 to '23. Having said that, v5.0 was in development well over 2 years I don't see how I can track down who all actually contributed. I could leave it out apart from the key contributors that I'm aware of and let others complain. I figured this was safer. As far as Amazon, they're going to a dateless format and asked me to remove the dates.
I see that Nanuk is 2021 - does it need an end date or update to '23?

Reply:
> I see that Nanuk is 2021 - does it need an end date or update to '23?

Doesn't really matter - I have no intentions of enforcing anything anyway

> As far as Amazon, they're going to a dateless format and asked me to remove the dates.

Well, based on what our lawyers told us when we started the project, a copyright without date is legally meaningless. You cannot claim a time-unlimited copyright - no such legal thing. Don't know if it invalidates the entire license, or if it just means Amazon has no legal rights. You should check with your own lawyers to see before releasing it that way.

Reply:
I've spoken to amazon guys, readding the date is fine too, so for consistency I'll do revert their dates.

Reply:
@rhc54 done